packages/frontend/amp/components/Elements.tsx

```diff
@@ -26,6 +26,7 @@ import { InteractiveUrl } from '@frontend/amp/components/elements/InteractiveUrl
 import { InteractiveMarkup } from '@frontend/amp/components/elements/InteractiveMarkup';
 import { MapEmbed } from '@frontend/amp/components/elements/MapEmbed';
 import { WithAds } from '@frontend/amp/components/WithAds';
+import { AudioAtom } from '@root/packages/frontend/amp/components/elements/AudioAtom';
```

Comment:
This is just how VS Code does imports (it's annoying I know) but we generally avoid @root and use @frontend/... instead.

Reply:
Updated